amisr/resolvedvelocities
# Python3 compatable

This modifies the code so it is both python 2 and python 3 comparable.  This PR also includes the ability to run resolvedvelocities from the command line, so it supersedes PR #7 .  This should address both #1  and #2.

To test:
1. Pull `python3_compatable` branch
```
git fetch
git checkout python3_compatable
```
2. Install
```
pip install .
```
3. Run with config file
```
resolvedvelocities config.ini
```
This should generate a test output hdf5 file.

Does call `resolvedvelocities` make sense?  Other options could be `resolvedvectors` or `resolvevectors`.  It's a minor point, but it's easy to fix at this point before this gets baked into batch processing scripts.
